.github/workflows/ci-pipeline.yaml

```diff
@@ -63,136 +35,67 @@ jobs:
           files_changed_data=$(curl -s --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -X GET -G "$URL")
           files_changed="$(echo $files_changed_data | jq -r '.[] | .filename')"
           # Adding || true to avoid "Process exited with code 1" errors
-          containers_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*" | sort | uniq || true)"
           flavors=($(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*/[^/]*/[^/]*" | sort | uniq || true))
-          container_name=$(echo "$containers_dirs_changed" | sed "s|bitnami/||g")
 
-          if [[ "${#flavors[@]}" -eq "1" ]]; then
-            # Changes done in only one container -> OK
-            echo "::set-output name=result::ok"
-            echo "::set-output name=container::${container_name}"
-            echo "::set-output name=path::${flavors[0]}"
-          elif [[ "${#flavors[@]}" -le "0" ]]; then
-            # Changes done in the containers/ folder but not inside a container subfolder -> Look for readme changes
-            readme_object="$(echo $files_changed_data | jq -r '[ .[] | select(.filename | test("^bitnami/[^/]+/README.md"))] | first')"
-            if [[ "${readme_object}" == "null" ]]; then
-              echo "::set-output name=error::No changes detected in containers. The rest of the tests will be skipped."
-              echo "::set-output name=result::skip"
```

Comment:
I'll have a look

Reply:
I think you are talking about the feature in charts to avoid running vib-verify when only `.md` files are changed. The code above do a different thing, but I've added added the feature to skip the verify phase in changes to `.md` files only

Reply:
Indeed I selected the wrong lines 😄 The change to skip the CI lgtm!